cmd/common/utils/flags.go

```diff
@@ -30,6 +31,69 @@ const (
 
 var SupportedOutputModes = []string{OutputModeColumns, OutputModeJSON, OutputModeCustomColumns}
 
+type CommonFlags struct {
```

Comment:
Maybe `BaseCommonFlags`? I don't know ... It seems strange to me to have a `CommonFlags` that has an embedded member also called `CommonFlags`, don't you think?